service/controller/v12/cluster_resource_set.go

```diff
@@ -109,13 +109,13 @@ func NewClusterResourceSet(config ClusterResourceSetConfig) (*controller.Resourc
 
 	var configMapResource controller.Resource
 	{
-		c := configmap.DefaultConfig()
-
-		c.CertSearcher = config.CertsSearcher
-		c.CloudConfig = cloudConfig
-		c.K8sClient = config.K8sClient
-		c.KeyWatcher = config.RandomkeysSearcher
-		c.Logger = config.Logger
+		c := configmap.Config{
```

Comment:
Boyscouting. I removed the default constructor of the config. Reason why this is also partially in `v12` is because my branch started a couple of days ago already. Meanwhile a new version got introduced. I ported my changes to `v13` but was too lazy to cleanup `v12` in that regard because it does not hurt and would only waste my time.